data-processing-lib/spark/src/data_processing_spark/runtime/spark/runtime_configuration.py

```diff
@@ -29,6 +32,15 @@ def __init__(
         super().__init__(transform_config=transform_config)
         self.runtime_class = runtime_class
 
+    def get_bcast_params(self, data_access_factory: DataAccessFactoryBase) -> dict[str, Any]:
+        """Allows retrieving and broadcasting to all the workers very large
+        configuration parameters, like the list of document IDs to remove for
+        fuzzy dedup, or the list of blocked web domains for block listing.  This
+        function is called after spark initialization, and before spark_context.parallelize()
```

Comment:
...called by the spark runtime after...

Reply:
Addressed in commit https://github.com/IBM/data-prep-kit/pull/687/commits/8d9bb663670ff4e55127de493e1eb9b40c2b2de0